packages/plugins/rubygem-foreman_google/rubygem-foreman_google.spec

```diff
@@ -0,0 +1,107 @@
+# template: foreman_plugin
+
+%global gem_name foreman_google
+%global plugin_name google
+%global foreman_min_version 3.5.0
+
+Name: rubygem-%{gem_name}
+Version: 0.0.2
+Release: 1%{?foremandist}%{?dist}
+Summary: Google Compute Engine plugin for the Foreman
+License: GPLv3
+URL: https://github.com/theforeman/foreman_google
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+Requires: ruby >= 2.5
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+Requires: ruby >= 2.5
+BuildRequires: ruby >= 2.5
+BuildRequires: rubygems-devel
+BuildArch: noarch
+Provides: foreman-plugin-%{plugin_name} = %{version}
+BuildRequires: rubygem(google-apis-compute_v1) >= 0.14
+BuildRequires: rubygem(google-apis-compute_v1) < 1
+BuildRequires: rubygem(google-cloud-compute) >= 0.2
+BuildRequires: rubygem(google-cloud-compute) < 1
+# end specfile generated dependencies
+
+# start package.json devDependencies BuildRequires
+BuildRequires: npm(@babel/core) >= 7.7.0
+BuildRequires: npm(@babel/core) < 8.0.0
+BuildRequires: npm(@theforeman/builder) >= 10.0.0
+BuildRequires: npm(@theforeman/builder) < 11.0.0
+# end package.json devDependencies BuildRequires
+
+# start package.json dependencies BuildRequires
+BuildRequires: npm(react-intl) >= 2.8.0
+BuildRequires: npm(react-intl) < 3.0.0
+# end package.json dependencies BuildRequires
```

Comment:
@stejskalleos If foreman_google has no React code, then all of these npm requires should not be needed. I think they are being generated from the plugin having a package.json -- which should be superfluous if there is no React UI as part of the plugin? Should we first trim up the plugin to have only what is needed before packaging it?

Reply:
I've been thinking about it, right now there is no React code, but that will change in the future as there are some things in UI that are let's say not ideal and would require some changes. But when (or if) it will happen, hard to say.

I lean towards to remove all the `*.js` stuff right now, we can add it later when necessary.

Reply:
So I found that we actually have some JS in the plugin - [webpack/legacy.js](https://github.com/theforeman/foreman_google/blob/develop/webpack/legacy.js). It's just one file, but it's there.